packages/amplify-codegen-e2e-tests/src/__tests__/feature-flags.test.ts

```diff
@@ -12,7 +12,6 @@ const codegenFeatureFlags = {
   retaincasestyle: true,
   generateindexrules: true,
   emitauthprovider: true,
-  enabledartnullsafety: true,
```

Comment:
Thanks for calling this out. So the process here is to first remove the E2E test from CLI repo, then merge the changes in codegen?

Reply:
Currently there is no formal way to completely remove the feature flags in CLI as that will cause missing flag value if you use the latest CLI on old projects which have the previous flag. The safer way is, instead of removing it from CLI, removing the usage of it in the downstream packages and update the doc as well for the ff deprecation.

Reply:
Now keeping the line `enabledartnullsafety: true,`